pandas/tests/io/json/test_ujson.py

```diff
@@ -654,7 +654,7 @@ def test_decode_big_escape(self):
         # Make sure no Exception is raised.
         for _ in range(10):
             base = '\u00e5'.encode("utf-8")
-            quote = compat.str_to_bytes("\"")
+            quote = b"\""
```

Comment:
would `b'"'` be clearer by avoiding the backslash?

Reply:
I would think so.